### ISSUE TITLE
Add Depth of field Blur

### ISSUE THREAD

**Issue by nvanderploeg** [andrewazores/archery-sight-simulator#10]
When aiming with focus at 70, The pin/aperture is blurry for most archers. Sometimes the target is blurry if they use the sight as the focal point, Would be cool to add blurs based on this

**Comment by andrewazores** [comment]
Good idea. There's probably some math out there that describes the type and "intensity" (for lack of a better generic term) of the blur for this scenario.
